.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,4 +40,7 @@ or AWS CLI commands to trigger something.
 - [ ] Fix linting errors
 - [ ] Make sure code coverage hasn't dropped
 - [ ] Provide verification config/commands/resources
-- [ ] Leave a comment that this is ready for review once you've finished the implementation
+- [ ] Change ready for review message below
+
+
+***Is this ready for review?:*** NO
```

Comment:
we can just have a check list for this:

- [ ] Ready for review -> NO
- [x] Ready for review -> YES